### DISCUSSION TITLE
Dependancies problem with wt package

### DISCUSSION THREAD

**Discussion by jm130794** [microsoft/vcpkg#39192] [General]
Hello,

I have a problem with the wt package. When I try to build my application, I get this error:

```
$ cmake -S . -B build
-- Configuring done
CMake Error at /vcpkg/scripts/buildsystems/vcpkg.cmake:598 (_add_executable):
  Target "syncadmdp" links to target "HPDF::HPDF" but the target was not
  found.  Perhaps a find_package() call is missing for an IMPORTED target, or
  an ALIAS target is missing?
Call Stack (most recent call first):
  CMakeLists.txt:8 (add_executable)


-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

Here is my CMakeLists.txt file : 

```
cmake_minimum_required(VERSION 3.22)

project(syncadmdp)

find_package(unofficial-libharu CONFIG REQUIRED)
find_package(wt CONFIG REQUIRED)

add_executable(syncadmdp
    src/main.cpp
)

target_include_directories(syncadmdp BEFORE PRIVATE
    ${CMAKE_CURRENT_SOURCE_DIR}/src/include
    ${CMAKE_CURRENT_BINARY_DIR}/src/include
)

target_link_libraries(syncadmdp PRIVATE
    Boost::filesystem
    Wt::Wt Wt::HTTP
    unofficial::libharu::hpdf
)
```
I couldn't find the package hpdf : 

 ```
/vcpkg/vcpkg search hpdf
The result may be outdated. Run `git pull` to get the latest results.
If your port is not listed, please open an issue at and/or consider making a pull request.  -  https://github.com/Microsoft/vcpkg/issues
```
Do you have any ideas?

Cheers.

**Comment by AugP** [comment]
HPDF seems to be part of the libharu library. I see you have "unofficial::libharu" in your CMakeLists.txt, are you using a custom version of that library?

**Reply by dg0yt** [reply]
This is what vcpkg `usage` suggests.

**Reply by dg0yt** [reply]
And `HPDF::HPDF` is an imported target created by `cmake/FindHPDF.cmake` in `wt` sources.
So probably an installation order bug for port `wt`

**Comment by grunc** [comment]
I had the same issue. I managed to make my project compile by:

-  adding glew, opengl and libharu as dependencies in my vcpkg manifest
- adding these lines before `find_package(wt CONFIG REQUIRED)`
`find_package(unofficial-libharu CONFIG REQUIRED)`
`find_package(OpenGL REQUIRED)`
`find_package(GLEW REQUIRED)`
`add_library(HPDF::HPDF ALIAS unofficial::libharu::hpdf)`
